common_sleep.Rmd

```diff
@@ -200,224 +230,271 @@ lattice::xyplot(Reaction ~ Days|Sx,
 611.9 & 9.61 \\
 9.61 & 35.08 
 \end{pmatrix}
-\end{pmatrix}
+\end{pmatrix} \tag{13}
 $$ 
-D: Random effects covariance matrix
-The model is built on the assumption that the b's come from a normal distribution with mean 0 and the residual variance in true intercept $\pi_{0i}$ across all individuals in the population of 611.9, the residual variance in true slope for all individuals is 35.08 and residual covariance between the intercept and the slope of 9.61.
-
-
 
 ```{r}
-sleep.o <- ranef(sleep.reml)$Subject
-head(sleep.o,10)
+D.sleep = unclass(VarCorr(sleep_reml))$Subject 
+pander::pander(D.sleep)
+
 ```
 
-This table preditcs the random effect for each subject.Almost all values lay within two standard devation from the mean.
+The model is built on the assumption that the b's come from a normal distribution with mean 0. The residual variance in true intercept $\pi_{0i}$ across all individuals in the population is 611.9, the residual variance in true slope $\pi_{1i}$ across all individuals in the population is 35.08 and the residual covariance between the true intercept $\pi_{0i}$ and the slope $\pi_{1i}$ is 9.61, as seen on the table above. 
+
+
+Next step is to check if the values retrieved are actually significant. We therefore check if the number of days have a significant effect on the reaction time.
+
+In the next chapter, we tested the fixed effects with Bootstrap and profile likelihood as the sample size was too small to use a Wald test. Next, we checked and compared different possible models using a likelihood ratio test.
 
-```{r}
 
-plot(sleep.o, main = "Random intercept (boi) vs random slope (b1i)")
-#plot modelled intercept and slope incl fixed effects
-ind.coef = coef(sleep.reml)$Subject
-head(ind.coef)
-interc.coef = ind.coef[,1]
-slope.coef = ind.coef[,2]
-plot(interc.coef,slope.coef, main = "Random intercept vs random slope 
-     (including fixed effects)")
-
-#plot individual intercept and slope to compare 
-intercept.lin.reg.coef2 = lin.reg.coef2[,1]
-slope.lin.reg.coef2 = lin.reg.coef2[,2]
-plot(intercept.lin.reg.coef2, slope.lin.reg.coef2, main = "Individual regression intercept vs slope")
+## Testing fixed effects - with bootstrap
+```{r echo=FALSE, message=FALSE, warning=FALSE}
+pander::pander(confint(sleep_reml,
+        method    = 'boot',
+        boot.type = 'perc',
+        oldNames  = F,
+        nsim      = 500))
+        
+        pander::pander(confint(sleep_reml,
+        method    = 'profile',
+        level     = 0.95,
+        oldNames  = F))
 ```
+The confidence intervals of the intercept and Days do not include 0. Therefore, both have a significant effect on the Reaction time.
+
+## Likelihood ratio test with Anova
 ```{r}
-ggplot(sleep.o) +
-  geom_point(aes(`(Intercept)`, Days))
+sleep.full      <- lmer(Reaction~1+Days+(1 + Days|Subject), REML = FALSE, data = sleep)
+sleep.intercept <- lmer(Reaction~1+(1|Subject), REML = FALSE, data = sleep)
+pander::pander(anova(sleep.full,sleep.intercept))
```

Comment:
why is this called tables six? The other tables are not numbered

Reply:
Yes I saw this too but its part of the automatic code so I dont know if you know how to fix it? Otherwise we have to keep it like that

Reply:
I think i can fix this within the latex code. But i'm not sure